### ISSUE TITLE
Model diverged with loss = NaN when training about 151K

### ISSUE THREAD

**Issue by guanfuchen** [loliverhennigh/Convolutional-LSTM-in-Tensorflow#10]
This is the tensorboard view, the model(residual_model) diverge when the training step is about 151K, what should I do to prevent model is diverged? Thank you for your implementing the convlstm, It help me a lot!!
![image](https://user-images.githubusercontent.com/22321977/33248043-8256d51e-d35d-11e7-859f-12a975275c7a.png)
